test/functional/lua/fs_spec.lua

```diff
@@ -70,7 +70,7 @@ describe('vim.fs', function()
 
   describe('dirname()', function()
     it('works', function()
-      local test_windows = package.config:sub(1, 1) == '\\'
+      local test_windows = vim.loop.os_uname().sysname == 'Windows_NT'
```

Comment:
In the tests we have a helper function `is_os`. Simply add

```lua
local is_os = helpers.is_os
```

to the top of the file (if it's not already there) and use it like this:

```lua
is_os('win')
```